pandas/core/indexing.py

```diff
@@ -449,6 +449,37 @@ def loc(self) -> _LocIndexer:
         8          4       5
         9          7       8
 
+        **Set values with a Series**
+
+        Set column values using a Series
```

Comment:
Maybe good to add to the end here `aligns along the index values of the Series and DataFrame`